react/components/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+export const MAX_TACHYONS_TOKEN = 11
+export const MIN_TACHYONS_TOKEN = 0
+
+export function getValidTachyonsTokenNumber(num: number = MIN_TACHYONS_TOKEN) {
+  const tokenValue = Math.min(Math.max(num, MIN_TACHYONS_TOKEN), MAX_TACHYONS_TOKEN)
+  return isNaN(tokenValue) ? MIN_TACHYONS_TOKEN : tokenValue
+}
```

Comment:
This is based on a recommendation from @klzns and I think it makes sense. 
You could rename this `utils` folder to `modules` and instead of having an `index.ts` file, have multiple files (or just one in this case), separating your functions by what they do. In this case it would be:
`modules/tachyonsToken.ts`, or something like that.
Also, this should not be inside your `components` folder, but in the root of `/react`.